corehq/apps/userreports/ui/forms.py

```diff
@@ -90,6 +90,12 @@ def clean_table_id(self):
             raise ValidationError(
                 _('Table id is too long. Your table id and domain name must add up to fewer than 40 characters')
             )
+        for src in self.instance.by_domain(self.domain):
+            if src.table_id == table_id and src.get_id != self.instance.get_id:
```

Comment:
i think our convention is mostly to use `_id` in python and `get_id` only in templates, but this isn't a big deal at all